charts/ccsm-helm/charts/zeebe/templates/service.yaml

```diff
@@ -19,5 +19,11 @@ spec:
     - port: {{ .Values.service.commandPort }}
       protocol: TCP
       name: {{ default "command" .Values.service.commandName }}
+    {{- range .Values.service.extraPorts }}
+    - port: {{ .port }}
+      targetPort: {{ .targetPort}}
+      protocol: TCP
+      name: {{ .name }}
+    {{- end }}
```

Comment:
:wrench: What about just introducing as variable `extraServices` and add it here via             
```
{{- if .Values.extraServices}}
            {{ .Values.extraServices | toYaml | nindent X }}
{{- end }}
```

Reply:
@Zelldon  not sure if I understood your question correctly. But as far as I see, I think you are suggesting that instead of appending ports to an existing service we give the user the ability to to create multiple services?

If ^^^ is the case, then I have already thought about this, however I concluded that I prefer to give the user a way to simply expose x ports to an existing service, simply because its easier/simpler for the user. 
If we allow users to add a service (and I think the feature will eventually be required), I would give the user more granular control over the service so that one can set very custom annotations, labels etc etc. Also for a custom service, global annotations may cause conflicts (e.g setting a domain via annotations) so i didnt want to cause any problems. 

Thats my logic :)

Reply:
Sorry I think I worded it wrongly :) What I of course meant is ports. So instead of having the logic to directly assign the properties here:
```
    - port: {{ .port }}
      targetPort: {{ .targetPort}}
      protocol: TCP
      name: {{ .name }}
```
We can just have as template:
```
{{- if .Values.extraServices}}
            {{ .Values.extraServices | toYaml | nindent X }}
{{- end }}
```
Similar to extraVolumnes, Mounts etc. in other components. I think it is not necessary to define here all properties in the template. 

As example https://github.com/camunda/camunda-cloud-helm/blob/main/charts/ccsm-helm/values.yaml#L248-L249 extraVolumes in the gateway values file and in the template we just copy them over https://github.com/camunda/camunda-cloud-helm/blob/main/charts/ccsm-helm/charts/zeebe-gateway/templates/gateway-deployment.yaml#L86-L88 if they exist